### ISSUE TITLE
 “Confirmation Error” popup displayed & user gets blocked to complete verification process with all possible ways for confirming phone number throughout the application         

### ISSUE THREAD

**Issue by ValoraQA** [valora-inc/wallet#1452]
**Frequency:** 100%
**Repro on build version:** Android internal build V1.22.0 & Test Flight Build V1.22.0 
**Does Not Repro on build version:** Play Store build V1.21.0 & Test Flight Build V1.21.0
**Repro on devices:**   iPhone 12(15), Redmi Note 8 (9.0), One Plus 7T(11.0), Redmi Note 8 (10.0), iPhone 6+ (12.5.4), iPhone SE (13.6.1),
**Repro Steps:**
1. Launch the application & redirect to the “Phone number” page
2. Enter a valid phone number by selecting a valid country code 
3. Now tap on enabled “Start” button
4. Complete RECAPTCHA page & now user should be on Confirming screen with 2 minutes timer
5. Wait for some time & observe that
**Current Behavior:** “Confirmation Error” popup displayed & user gets blocked to Confirm phone number while creating an account
**Expected Behavior:** 2 minutes timer should be completed & user should be able to confirm phone number by entering received 3 codes without any error message 
**Investigation:** Also we have observed the same issue when trying to confirm phone numbers from the below ways/features:
                           1. Restoring Account
                           2. In-app CTA
                           3. Supercharge your crypto! CTA
                           4. Settings
**Impact:** User gets blocked to Confirm phone number using any option throughout an application
**Attachment:** [ConfirmationErrorForVerification.mp4](https://drive.google.com/file/d/1DHsJgnbU6TcBTWfdx66JQ5BM7lcLcrPb/view?usp=sharing)

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D581](https://valora.testquality.com/project/12128/result/2571308/defect/9136).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D582](https://valora.testquality.com/project/12128/result/2571312/defect/9137).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D588](https://valora.testquality.com/project/12128/result/2571306/defect/9144).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D589](https://valora.testquality.com/project/12128/result/2571411/defect/9145).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D590](https://valora.testquality.com/project/12128/result/2571412/defect/9146).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D591](https://valora.testquality.com/project/12128/result/2571413/defect/9147).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D592](https://valora.testquality.com/project/12128/result/2571414/defect/9148).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D593](https://valora.testquality.com/project/12128/result/2571415/defect/9149).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D594](https://valora.testquality.com/project/12128/result/2571418/defect/9150).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D595](https://valora.testquality.com/project/12128/result/2571420/defect/9151).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D596](https://valora.testquality.com/project/12128/result/2571419/defect/9152).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D597](https://valora.testquality.com/project/12128/result/2571438/defect/9153).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D598](https://valora.testquality.com/project/12128/result/2571442/defect/9154).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D601](https://valora.testquality.com/project/12128/result/2571311/defect/9162).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D602](https://valora.testquality.com/project/12128/result/2571401/defect/9163).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D603](https://valora.testquality.com/project/12128/result/2571402/defect/9164).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D604](https://valora.testquality.com/project/12128/result/2571403/defect/9165).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D605](https://valora.testquality.com/project/12128/result/2571404/defect/9166).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D606](https://valora.testquality.com/project/12128/result/2571405/defect/9167).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D607](https://valora.testquality.com/project/12128/result/2571406/defect/9168).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D608](https://valora.testquality.com/project/12128/result/2571407/defect/9169).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D609](https://valora.testquality.com/project/12128/result/2571408/defect/9170).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D610](https://valora.testquality.com/project/12128/result/2571409/defect/9171).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D611](https://valora.testquality.com/project/12128/result/2571410/defect/9172).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D612](https://valora.testquality.com/project/12128/result/2571430/defect/9173).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D613](https://valora.testquality.com/project/12128/result/2571434/defect/9174).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D618](https://valora.testquality.com/project/12128/result/2571391/defect/9223).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D621](https://valora.testquality.com/project/12128/result/2571392/defect/9226).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D622](https://valora.testquality.com/project/12128/result/2571393/defect/9227).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D623](https://valora.testquality.com/project/12128/result/2571394/defect/9228).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D624](https://valora.testquality.com/project/12128/result/2571395/defect/9229).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D625](https://valora.testquality.com/project/12128/result/2571397/defect/9230).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D626](https://valora.testquality.com/project/12128/result/2571398/defect/9231).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D627](https://valora.testquality.com/project/12128/result/2571399/defect/9232).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D628](https://valora.testquality.com/project/12128/result/2571400/defect/9233).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D629](https://valora.testquality.com/project/12128/result/2573711/defect/9234).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D630](https://valora.testquality.com/project/12128/result/2573712/defect/9235).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D631](https://valora.testquality.com/project/12128/result/2573713/defect/9236).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D632](https://valora.testquality.com/project/12128/result/2573714/defect/9237).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D633](https://valora.testquality.com/project/12128/result/2573716/defect/9238).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D634](https://valora.testquality.com/project/12128/result/2573717/defect/9239).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D635](https://valora.testquality.com/project/12128/result/2573747/defect/9240).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D636](https://valora.testquality.com/project/12128/result/2573748/defect/9241).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D637](https://valora.testquality.com/project/12128/result/2573749/defect/9242).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D639](https://valora.testquality.com/project/12128/result/2573775/defect/9244).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D640](https://valora.testquality.com/project/12128/result/2573777/defect/9245).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D641](https://valora.testquality.com/project/12128/result/2573778/defect/9246).

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using Latest Android Internal Build V1.22.1, Test flight build V1.22.0 & observed that the issue is fixed and user is able to complete phone number verification successfully. Hence we are closing this issue.

**Devices:** iPhone 12(15.0), Samsung Galaxy F41 (11.0), Redmi Note 8 (9.0), One plus 7t(11.0)
**Attachment:** [Phone number Verification Successful.mp4](https://drive.google.com/file/d/1tpe6AZQM7J4xyzJFS5plAO7-MU2Pn36B/view?usp=sharing)
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using Latest android internal build V1.22.1 & observed this issue is still reproducible. Hence we are re-opening the issue.

**Repro on Devices:** Google Pixel 2XL (11.0), Samsung Galaxy A70 (9.0), Google Pixel XL(10.0)
**Does not Repro Devices:** iPhone 12 (15.0), One plus 7T(11.0), Redmi Note 8 (9.0)
**Attachment:** [Confirmation error.mp4](https://drive.google.com/file/d/18vNf4F_npods-W4-GgvnvciIWZbWb6rs/view?usp=sharing)
Thanks..!!

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D675](https://valora.testquality.com/project/12128/result/2581217/defect/9319).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D676](https://valora.testquality.com/project/12128/result/2581341/defect/9320).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D677](https://valora.testquality.com/project/12128/result/2581342/defect/9321).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D678](https://valora.testquality.com/project/12128/result/2581343/defect/9322).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D679](https://valora.testquality.com/project/12128/result/2581344/defect/9323).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D680](https://valora.testquality.com/project/12128/result/2581345/defect/9324).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D681](https://valora.testquality.com/project/12128/result/2581347/defect/9325).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D682](https://valora.testquality.com/project/12128/result/2581348/defect/9326).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D683](https://valora.testquality.com/project/12128/result/2581349/defect/9327).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D686](https://valora.testquality.com/project/12128/result/2581224/defect/9330).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D691](https://valora.testquality.com/project/12128/result/2581346/defect/9340).

**Comment by ValoraQA** [comment]
https://valora.testquality.com defect [D698](https://valora.testquality.com/project/12128/result/2586714/defect/9357).

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using Latest Android Internal Build V1.22.4, Test flight build V1.22.4, App store build V1.22.4  & observed that the issue is fixed now. User is able to complete phone number verification successfully. Hence we are closing this issue.
**Note:** Currently Latest Play store build displayed is V1.21.0, So we have verified this issue in Latest Android Internal Build only. 

**Devices:** iPhone 13 mini (15.1.1), Google Pixel 2XL (11.0), Redmi Note 8 (10.0), iPhone 12 (14.7.1)
**Attachment:** [Phone Number Confirmation Complete.mp4](https://drive.google.com/file/d/1532EvdxQ7Cj45nfh5S7lJiQvFzoDQN4t/view?usp=sharing)
Thanks..!!
